apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -14,48 +16,59 @@ import {
   ONE_TIME_DIVIDEND,
   RECURRING_DIVIDEND,
   ONE_TIME_MERIT,
+  READY,
+  CLAIMED,
 } from '../../utils/constants'
 import { Empty } from '../Card'
 import Metrics from './Metrics'
-import { useAppState } from '@aragon/api-react'
+import { useAppState, useNetwork } from '@aragon/api-react'
 import BigNumber from 'bignumber.js'
-import { displayCurrency, getStatus } from '../../utils/helpers'
+import { displayCurrency, getStatus, getStatusId } from '../../utils/helpers'
 
 const MyRewards = ({
   myRewards,
   myMetrics,
   viewReward,
   claimReward,
+  claimHashes,
 }) => {
+  const network = useNetwork()
   const rewardsEmpty = myRewards.length === 0
 
+  const getEtherscanLink = reward => {
+    const networkChunk = network.id === 1 ? '' : `${network.type}.`
+    const claimHash = claimHashes[reward.rewardId]
+    const link = `https://${networkChunk}etherscan.io/tx/${claimHash}`
+    return link
+  }
+
   if (rewardsEmpty) {
     return <Empty noButton />
   }
 
-  const renderMenu = (reward) => (
-    <ContextMenu>
-      <StyledContextMenuItem
-        onClick={() => viewReward(reward)}
-      >
-        <IconView css={{
-          marginRight: '11px',
-          marginBottom: '2px',
-        }}/>
-        View
-      </StyledContextMenuItem>
-      {(reward.claims < reward.disbursements.length && (reward.disbursements[reward.claims] < Date.now())) && (
-        <StyledContextMenuItem
-          onClick={() => claimReward(reward)}
-        >
-          <IconCoin css={{
-            marginRight: '11px',
-            marginBottom: '2px',
-          }}/>
-        Claim
-        </StyledContextMenuItem>)}
-    </ContextMenu>
-  )
+  const renderMenu = (reward) => {
```

Comment:
We could also move this outside of the component, and make it a component. But that's probably out-of-scope for now.

Reply:
So `renderMenu` is used as `DataView`'s `renderEntryActions` property. According to AragonUI's [docs](https://github.com/aragon/aragon-ui/tree/master/src/components/DataView#renderentryactionsentry-index--selected-mode-), `renderEntryActions` gets a function, not a component.
![Screenshot from 2019-11-27 12-19-46](https://user-images.githubusercontent.com/16065447/69745584-8105a080-1110-11ea-83fe-988dd80509e7.png)

Does it still make sense for it to become a component?

Reply:
I still think it makes sense to move it outside. I guess we can't make it a component right now, though it might be worth opening an issue in the aragonUI repo.